config/config.yml

```diff
@@ -503,3 +503,7 @@ STAT_COUNTER_BATCH_SIZE: 100
 # Number of hours to retain the original creator for works that have been
 # orphaned.
 ORIGINAL_CREATOR_TTL_HOURS: 72
+
+# Regex for the site's domains, used to validate that user-supplied
+# links are located on the Archive when appropriate.
+APP_URL_REGEX: "^(https?:\\/\\/)?(www\\.|(insecure\\.))?(archiveofourown|ao3)\\.(org|com).*"
```

Comment:
I went to `archiveofourown.com` and noticed that it has a proxy warning; should it be added to the allowed hosts, or do we want to remove it from this regex? Similarly, `ao3.com` does not resolve at all

Reply:
Yeah, let's add `archiveofourown.com` (and `archiveofourown.net`, which is also displaying the proxy notice) to `PERMITTED_HOSTS`. 

I think `ao3.org` is the only `ao3` we have, no `.com` or `.net`, but we can check with Systems.

Reply:
Just a heads up if you didn't see the convo, only ao3.org is ours.